.claude/commands/resolve.md

```diff
@@ -31,9 +31,11 @@ Automatically fetch and address PR review comments. This command examines review
 
 1. **Auto-detect PR context**:
 
-   - Use `gh pr view --json url -q '.url'` to get PR info for the current branch
-   - Parse to extract owner, repo, and PR number
-   - If auto-detection fails, inform the user that no PR was found for the current branch
+   - First check for environment variables:
+     - If `PR_NUMBER` and `GITHUB_REPOSITORY` are set, read them and parse `GITHUB_REPOSITORY` as `owner/repo` and use `PR_NUMBER` directly
```

Comment:
`GITHUB_REPOSITORY` is automatically set by GitHub Actions, but `PR_NUMBER` is not:

https://docs.github.com/en/actions/reference/workflows-and-actions/variables